webview-ui/src/utils/__tests__/commandPattern.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { extractCommandPattern, formatCommandPatternForDisplay } from "../commandPattern"
+
+describe("commandPattern", () => {
+	describe("extractCommandPattern", () => {
+		it("should extract base command patterns correctly", () => {
+			expect(extractCommandPattern("wc -l foo.txt")).toBe("wc -l")
+			expect(extractCommandPattern("cd /path/to/project && npm install")).toBe("cd && npm install")
+			expect(extractCommandPattern("git status")).toBe("git status")
+			expect(extractCommandPattern("ls -la /some/path")).toBe("ls -la")
+			expect(extractCommandPattern("npm test -- --coverage")).toBe("npm test")
```

Comment:
so this needs refinement, especially the `cd &&` stuff, but I do think we need something to generalize